test/tests/restart/restart_add_variable/add_variable_restart.i

```diff
@@ -117,12 +117,22 @@
   num_steps = 3
 []
 
+[Reporters/data]
+  type = RestartableDataReporter
+  include = 'FEProblemBase/*/equation_systems'
+  execute_on = initial
+[]
+
 [Outputs]
   [./out]
     type = Exodus
     elemental_as_nodal = true
     execute_elemental_on = none
   [../]
+  [json]
+    type = JSON
+    execute_system_information_on = none
```

Comment:
```suggestion
    execute_system_information_on = none
    execute_on = INITIAL
```

We only load the data at the beginning of the solve - thus this value shouldn't change on timesteps (look at the json output, the data is repeated for each timestep). This `execute_on` input option controls when an object will run. With this, it only outputs at the beginning (which is all we care about!)

This will require that you "regold" (change the file in gold) the json file